static/app/components/menuListItem.tsx

```diff
@@ -96,6 +109,7 @@ function BaseMenuListItem({
   trailingItems = false,
   trailingItemsSpanFullHeight = false,
   isFocused = false,
+  isSelected = false,
```

Comment:
`isSelected` isn't directly consumed by `MenuListItem` (it doesn't affect the component's styles), but it's still a useful state to be passed to the `leading`/`trailingItems` render props.